src/pages/MuteerOverview/MuteerOverview.tsx

```diff
@@ -0,0 +1,398 @@
+    BeleidsprestatiesRead,
+    BeleidsmodulesRead,
+    BeleidsdoelenRead,
+    MaatregelenRead,
+    ThemasRead,
+    VerordeningenRead,
+} from '@/api/fetchers.schemas'
+import { ContainerMain } from '@/components/Container'
+import Dropdown from '@/components/Dropdown'
+import { LoaderCard } from '@/components/Loader'
+import PageSpecificNavBar from '@/components/PageSpecificNavBar'
+import { PopUpAddPolicyToModule } from '@/components/Popup'
+import { filteredDimensieConstants } from '@/constants/dimensies'
+import useAuth from '@/hooks/useAuth'
+import { getFetcherForType } from '@/utils/getFetchers'
+import { removePolicyFromModule } from '@/utils/removePolicyFromModule'
+
+/**
+ * A component to display all the objects from a specific dimension
+ * @param {Object} dimensieConstants - Contains the variables of the dimension
+ */
+
+type PossiblePolicyTypes =
+    | AmbitiesRead
+    | BelangenRead
+    | BeleidskeuzesRead
+    | BeleidsregelsRead
+    | BeleidsprestatiesRead
+    | BeleidsmodulesRead
+    | BeleidsdoelenRead
+    | MaatregelenRead
+    | ThemasRead
+    | VerordeningenRead
+
+interface MuteerOverviewProps {
+    dimensieConstants: filteredDimensieConstants
+    hideAddObject?: boolean
+}
+
+const MuteerOverview = ({
+    dimensieConstants,
+    hideAddObject,
+}: MuteerOverviewProps) => {
+    const { user } = useAuth()
+
+    const [filterQuery, setFilterQuery] = useState('')
+    const [isLoading, setIsLoading] = useState(true)
+    const [ascending, setAscending] = useState(true)
+    const [policyObjects, setPolicyObjects] = useState<PossiblePolicyTypes[]>(
+        []
+    )
+
+    const navigate = useNavigate()
+
+    const titleSingular = dimensieConstants.TITLE_SINGULAR
+    const titlePlural = dimensieConstants.TITLE_PLURAL
+    const createNewSlug = dimensieConstants.SLUG_CREATE_NEW
+    const overviewSlug = dimensieConstants.SLUG_OVERVIEW
+
+    const useGetLineage = getFetcherForType(titleSingular)
+    const { data: policyObjectsFromAPI } = useGetLineage()
+
+    /**
+     * When the component mounts, fetch the objects from the API and prepare state:
+     * 1. Sort based on ascending or descending
+     * 2. Filter out archived objects
+     * 3. Filter based on filter Query value
+     * 4. Set in state
+     */
+    useEffect(() => {
+        if (!policyObjectsFromAPI) return
+
+        const sortedAndFilteredPolicyObjects = policyObjectsFromAPI
+            .sort((a, b) =>
+                ascending
+                    ? a.Titel! > b.Titel!
+                        ? 1
+                        : -1
+                    : a.Titel! < b.Titel!
+                    ? 1
+                    : -1
+            )
+            .filter(
+                policy =>
+                    policy.Eind_Geldigheid &&
+                    isValid(new Date(policy.Eind_Geldigheid)) &&
+                    isBefore(new Date(), new Date(policy.Eind_Geldigheid))
+            )
+            .filter(policyObject =>
```

Comment:
@Aidenbuis You can add this part in the previous filter right?

Reply:
Extracted in utility, see: 2b5e7ee00e7266d96f4f22e13c965a49edf87c47